WORKSPACE

```diff
@@ -83,3 +83,14 @@ filegroup(
 
 load("@io_bazel_rules_scala//scala:toolchains.bzl","scala_register_toolchains")
 scala_register_toolchains()
+
+load("//scala:scala_maven_import_external.bzl", "java_import_external")
+java_import_external(
+    name = "com_google_guava_guava_21_0",
+    licenses = ["notice"],  # Apache 2.0
+    jar_urls = [
```

Comment:
are these alternative urls for the same sha?

Reply:
I can mirror files if you need it. Not sure I follow.